### DISCUSSION TITLE
Afrikaans Language Leader

### DISCUSSION THREAD

**Discussion by nicocoetzee** [OHF-Voice/intents#1169] [Language Leaders]
Hi,

I would like to apply for the position as Afrikaans language leader. I’m a native Afrikaans speaker.

I look forward to contributing! 

Many thanks,
n.

**Comment by BinaryDynamics** [comment]
Hi Nico,

I'm also willing to contribute if you need help.

Johan

**Comment by LiveBootleg** [comment]
Nico,

I also want to volunteer to help with the Afrikaans language, let me know where I can support

Thanks
Pieta

**Comment by synesthesiam** [comment]
PR by @wernerhp has been merged: https://github.com/home-assistant/intents/pull/1431

Sorry for the delay in responding here. For anyone wanting to be a language leader, please submit 1 or 2 small PRs that I can take a look at :slightly_smiling_face:

**Reply by GSVil** [reply]
Hi Michael.

Don't judge please. 😄 I've never used github other than download packages, or read documentation etc. I would like to see if I can contribute to Afrikaans.  I don't want to be an expert using github, I just want to help with the language, so the minimum information on how to use github to contribute would be great. I'm sure I can figure it out, but how do I start....do I  fork a single file and modify it, and then submit it for review?

**Reply by synesthesiam** [reply]
No judgement :slightly_smiling_face: 
You can fork this repo, modify a single file, and then submit a pull request for review. Just make sure to tag me :+1:

**Reply by GSVil** [reply]
Thankyou...
